src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBChainTest.java

```diff
@@ -79,9 +79,9 @@ void test() {
 
         // assert
         assertThat(logAggregator.messages, is(ImmutableList.of(
-            "method=createTable(), table=ctx1.mt_sharedtablestatic_s_nolsi, {AttributeDefinitions: [{AttributeName: hk,AttributeType: S}, {AttributeName: gsi_s_hk,AttributeType: S}, {AttributeName: gsi_s_s_hk,AttributeType: S}, {AttributeName: gsi_s_s_rk,AttributeType: S}, {AttributeName: gsi_s_n_hk,AttributeType: S}, {AttributeName: gsi_s_n_rk,AttributeType: N}, {AttributeName: gsi_s_b_hk,AttributeType: S}, {AttributeName: gsi_s_b_rk,AttributeType: B}],TableName: ctx1.mt_sharedtablestatic_s_nolsi,KeySchema: [{AttributeName: hk,KeyType: HASH}],GlobalSecondaryIndexes: [{IndexName: gsi_s,KeySchema: [{AttributeName: gsi_s_hk,KeyType: HASH}],Projection: {ProjectionType: ALL,},ProvisionedThroughput: {ReadCapacityUnits: 1,WriteCapacityUnits: 1}}, {IndexName: gsi_s_s,KeySchema: [{AttributeName: gsi_s_s_hk,KeyType: HASH}, {AttributeName: gsi_s_s_rk,KeyType: RANGE}],Projection: {ProjectionType: ALL,},ProvisionedThroughput: {ReadCapacityUnits: 1,WriteCapacityUnits: 1}}, {IndexName: gsi_s_n,KeySchema: [{AttributeName: gsi_s_n_hk,KeyType: HASH}, {AttributeName: gsi_s_n_rk,KeyType: RANGE}],Projection: {ProjectionType: ALL,},ProvisionedThroughput: {ReadCapacityUnits: 1,WriteCapacityUnits: 1}}, {IndexName: gsi_s_b,KeySchema: [{AttributeName: gsi_s_b_hk,KeyType: HASH}, {AttributeName: gsi_s_b_rk,KeyType: RANGE}],Projection: {ProjectionType: ALL,},ProvisionedThroughput: {ReadCapacityUnits: 1,WriteCapacityUnits: 1}}],ProvisionedThroughput: {ReadCapacityUnits: 1,WriteCapacityUnits: 1},StreamSpecification: {StreamEnabled: true,StreamViewType: NEW_AND_OLD_IMAGES}}",
```

Comment:
An extra comma is added because the model class for the dynamodb table description changes when global tables were added